app/scripts/controllers/preferences.test.js

```diff
@@ -243,74 +243,79 @@ describe('preferences controller', () => {
     });
   });
 
-  describe('setOpenSeaEnabled', () => {
-    it('should default to false', () => {
+  describe('setOpenSeaEnabled', function () {
+    it('should default to false', function () {
       expect(
         preferencesController.store.getState().openSeaEnabled,
       ).toStrictEqual(false);
     });
 
-    it('should set the openSeaEnabled property in state', () => {
+    it('should set the openSeaEnabled property in state', function () {
       preferencesController.setOpenSeaEnabled(true);
       expect(
         preferencesController.store.getState().openSeaEnabled,
       ).toStrictEqual(true);
     });
   });
 
-  describe('setAdvancedGasFee', () => {
-    it('should default to null', () => {
+  describe('setAdvancedGasFee', function () {
+    it('should default to null', function () {
```

Comment:
Updated test description:
```suggestion
    it('should default to an empty object', function () {
```